lib/uinta/plug.ex

```diff
@@ -143,6 +143,7 @@ if Code.ensure_loaded?(Plug) do
         query: query(graphql_info, opts),
         status: Integer.to_string(conn.status),
         timing: formatted_diff(diff),
+        duration_ms: diff,
```

Comment:
I was reading `formatted_diff` and it looks like this `diff` number is in microseconds. Would it make sense to either convert to `ms` or change the name of the field to `_us`?

Reply:
oh yea you are right, for some reason I got it opposite. I I think converting to ms is good. The us second is not that common for most request :)

Reply:
Awesome, let's do it 👍